lib/jira/util/index.js

```diff
@@ -6,16 +6,20 @@ module.exports = function (jiraClient) {
       ...issueMap,
       [issue.key]: issue
     }), {})
-
     return text.replace(jiraIssueReferenceRegex, (match, issueKey) => {
-      if (!issueMap[issueKey]) {
+      if (!issueMap[issueKey] || isLinkified(match)) {
```

Comment:
FWIW it might be easier to just append a [reference link] to the bottom of the text rather than replace inline.

[reference link]: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links